src/test/regress/sql/with_basics.sql

```diff
@@ -156,6 +156,44 @@ FROM
 JOIN
   users_table USING (user_id);
 
+-- column aliases in CTE
+WITH top_ten(id, val1) AS (
```

Comment:
Well, I should have added this in #1876, but I didn't think that we could write such queries:
```SQL
 SELECT * FROM (SELECT user_id, value_1, value_2 from users_table OFFSET 0) as foo(x, y) ORDER BY 1 DESC, 2 DESC, 3 DESC LIMIT 5;
```

Would you mind adding this to `src/test/regress/sql/subquery_complex_target_list.sql` ?

Reply:
added

Reply:
done